taglib/toolkit/trefcounter.cpp

```diff
@@ -66,7 +66,7 @@ namespace TagLib
     RefCounterPrivate() :
       refCount(1) {}
 
-    volatile ATOMIC_INT refCount;
+    ATOMIC_INT refCount;
```

Comment:
But this would remove `volatile` for all the other cases (`HAVE_GCC_ATOMIC, HAVE_WIN_ATOMIC, HAVE_MAC_ATOMIC, HAVE_IA64_ATOMIC`) too, which is not intended. Wouldn't it be better to remove `volatile` only in the `else` part, i.e. in [line 55 of trefcounter.cpp](https://github.com/taglib/taglib/blob/4971f8fb032f0cde8c2437cea44b6217b0f37269/taglib/toolkit/trefcounter.cpp#L55)?

Reply:
Yep. Fixed